python/lsst/eochar/pixel_covar.py

```diff
@@ -0,0 +1,106 @@
+"""
+Pixel covariance calculator
+
+Compute the pixel covariance matrix on a pair of
+flat fields, taking masks into account.
+If (k,l) is the (row,col) position of a pixel,
+its pixel covariance is
+Mean(Diff(j,i)*Diff(j-l,i-k)) - Mean(Diff(j,i))*Mean(Diff(j-l,i-k))
+where Diff stands for the (flat1-flat2) array, and "*" in the
+first term is an element-wise product.
+"""
+
+from lsst.eotest.sensor.MaskedCCD import MaskedCCD
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import numpy as np
+from math import sqrt
```

Comment:
I would use np.sqrt instead of math.sqrt below, so this import statement can be omitted.